cmake/FindOpenSCAP.cmake

```diff
@@ -0,0 +1,41 @@
+SET(OpenSCAP_POSSIBLE_ROOT_DIRS
+    "${OPENSCAP_ROOT_DIR}"
+    "$ENV{OPENSCAP_ROOT_DIR}"
+    "$ENV{ProgramFiles}"
+    "/usr/local/share"
+    "/usr/share/"
+    "/usr/local"
+    "/usr"
+    "/usr/bin"
+    "/usr/sbin"
+    "/opt"
+    "/opt/local"
+  )
+
+FOREACH(NAME ${OpenSCAP_POSSIBLE_ROOT_DIRS})
+    FIND_PATH(OpenSCAP_XSL_DIR NAMES xccdf_1.1_to_1.2.xsl
+    PATHS "${NAME}"
+    PATH_SUFFIXES "share/openscap/xsl/"
+  )
+ENDFOREACH()
+
+FOREACH(NAME ${OpenSCAP_POSSIBLE_ROOT_DIRS})
+    FIND_PROGRAM(OSCAP NAMES oscap
+    PATHS "${NAME}"
+    PATH_SUFFIXES "bin/"
+  )
+ENDFOREACH()
+
+IF (NOT EXISTS "${OpenSCAP_XSL_DIR}")
+    MESSAGE(SEND_ERROR
+            "ERROR: The OpenSCAP XSL directory was not found. Please specify the OpenSCAP ROOT DIR with the OPENSCAP_ROOT_DIR environment variable.")
+ELSE()
+    SET(OSCAP_XCCDF_XSL_1_2 "${OpenSCAP_XSL_DIR}/xccdf_1.1_to_1.2.xsl")
+ENDIF()
+
+IF (NOT EXISTS "${OSCAP}")
+    MESSAGE(SEND_ERROR
+            "ERROR: The OpenSCAP executable was not found. Please specify the OpenSCAP ROOT DIR with the OPENSCAP_ROOT_DIR environment variable.")
+ELSE()
+    SET(OSCAP_EXECUTABLE "${OSCAP}")
```

Comment:
this is unnecessary, just use the new variable... And keep the convention, prefix all variables coming out of the find script with OPENSCAP.